docs/embedding/static-embedding-parameters.md

```diff
@@ -165,6 +165,17 @@ You can preview appearance settings from your question or dashboard's [embedded
 
 \* Available on [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.metabase.com/product/enterprise) plans.
 
+### Download results
```

Comment:
```suggestion
### Allow people to download the results of an embedded question
```

Slightly more specific, and frames it from the perspective of an embedder.

Otherwise maybe just:

> ### Download results of an embedded question.